cuenca_validations/types/requests.py

```
@@ -547,13 +547,27 @@ class SessionRequest(BaseRequest):
     failure_url: Optional[SerializableAnyUrl] = None
+    resource_id: Optional[str] = None
+
+    @model_validator(mode='before')
+    @classmethod
+    def validate_metadata(cls, values: dict) -> dict:
+        if values[
+            'type'
+        ] == SessionType.metamap_verification and not values.get(
+            'resource_id'
+        ):
+            raise ValueError('Resource id expected for this session')
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)